File: features/transaction/buyerorder/src/main/java/com/tokopedia/buyerorder/detail/data/ConditionalInfo.java
package com.tokopedia.buyerorder.detail.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ConditionalInfo {
    @SerializedName("text")
    @Expose
    private String text;
    @SerializedName("color")
    @Expose
    private Color color;

    public String text() {
        return text;
    }

    public Color color() {
        return color;
    }

    @Override
    public String toString() {
        return "ConditionalInfo{"
                + "text=" + text + ", "
                + "color=" + color
                + "}";
    }
}

File: features/transaction/buyerorder/src/main/java/com/tokopedia/buyerorder/detail/data/ConditionalInfo.kt
package com.tokopedia.buyerorder.detail.data

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

data class ConditionalInfo (
    @SerializedName("text")
    @Expose
    val text: String = "",

    @SerializedName("color")
    @Expose
    val color: Color = Color()
)

